.github/workflows/ghcr-cleaner.yml

```diff
@@ -0,0 +1,29 @@
+name: Github Container Registry Cleaner
+
+on:
+  schedule:
+    - cron: "*/5 * * * *"
```

Comment:
maybe to much?

Reply:
yeah for now it is too much, i would change this to every week for the "production" setting